File: MODUL.3/PRAK301_2310817120010_NurHikmah.py
satu, dua = map(int, input('Input\n').split())

if satu <= dua:
    kecil = satu
    besar = dua
else:
    kecil = dua
    besar = satu

print("\nOutput")
print(f"{int(kecil)} {int(besar)}")

File: MODUL.3/PRAK301_2310817120010_NurHikmah.c
#include <stdio.h>

int main()
{
    float satu, dua, kecil, besar;

    printf("Input\n");
    scanf("%f %f", &satu, &dua);

    if (satu <= dua)
    {
        kecil = satu;
        besar = dua;
    }
    else
    {
        kecil = dua;
        besar = satu;
    }
    printf("\nOutput \n%.f %.f", kecil, besar);

    return 0;
}